app/uk/gov/hmrc/propertylinking/errorhandler/CustomErrorHandler.scala

```diff
@@ -53,12 +59,20 @@ class CustomErrorHandler @Inject()()(implicit override val messagesApi: Messages
       _.split("-")(2)
     }
 
+  override def onClientError(request: RequestHeader, statusCode: Int, message: String): Future[Result] =
+    statusCode match {
+      case FORBIDDEN => Future.successful(Forbidden(forbiddenErrorTemplate(request)))
+      case _         => super.onClientError(request, statusCode, message)
+    }
+
   override def onServerError(request: RequestHeader, exception: Throwable): Future[Result] =
     exception match {
       case error: AuthorisationFailure =>
         logger.info(s"business rates authorisation returned ${error.message}, redirecting to login.")
         Future.successful(
           Redirect(appConfig.basGatewaySignInUrl, Map("continue_url" -> Seq(request.uri), "origin" -> Seq("voa"))))
+      case upstreamEx: UpstreamErrorResponse if upstreamEx.statusCode == FORBIDDEN =>
+        Future.successful(Forbidden(forbiddenErrorTemplate(request)))
```

Comment:
I suppose we really only need one of these 403 handlers, but I'm not sure which one.